plutus-playground-server/usecases/Game.hs

```diff
@@ -63,12 +63,12 @@ gameInstance = Scripts.validator @Game
 
 -- create a data script for the guessing game by hashing the string
 -- and lifting the hash to its on-chain representation
-hashString :: String -> HashedString
+hashString :: Prelude.String -> HashedString
```

Comment:
Perhaps it's too much for this PR, but it would be nice to establish some best-practices over how to do this and stick to them (and document them!). Perhaps:
```
import qualified Prelude as Haskell
import qualified PlutusTx.Prelude as Plutus
```
? And if you're writing a module that's predominantly one or the other you can drop the qualification.

Reply:
I have fixed the existing `import qualified Prelude` imports.